front-end/src/App.test.js

```diff
@@ -27,15 +27,15 @@ describe('App tests', () => {
         store.dispatch(setTranscriptUploadStatusTrue())
         renderComponent();
         userEvent.click(screen.getByText('View Analysis'));
-        expect(await screen.getByTestId('analysis-page')).toBeInTheDocument();
-        expect(await screen.queryByTestId('main-page')).not.toBeInTheDocument();
+        expect(screen.getByTestId('analysis-page')).toBeInTheDocument();
+        expect(screen.queryByTestId('main-page')).not.toBeInTheDocument();
```

Comment:
not anything bad to say about the code, I just would like to know, what's the difference between getByTestId and queryByTestId and when do you know which one to use?

Reply:
getByTestID assumes that something exists, and if it does not, throws an error

VS

query which makes no such assumption.

So we use query when we are testing for lack of existence and get for existence.

Why not use query for both? Because VScode told me not to and I trust the computer gods.